src/website/app/demos/Table/examples/sortablePaginated.js

```diff
@@ -28,60 +30,69 @@ active sort, current page, and page size. Note that the
           super(props);
 
           this.state = {
-            data: this.props.data,
             currentPage: 1,
-            sort: undefined
+            sort: { key: 'name' }
           };
 
           this.handleSort = this.handleSort.bind(this);
           this.handlePageChange = this.handlePageChange.bind(this);
+          this.sort = this.sort.bind(this);
         }
 
         handleSort(sort) {
-          const sortedData = data.slice(0).sort((a,b) => {
-            const asc = defaultSortComparator(a,b,sort.key);
-            const desc = asc * -1;
-            return sort.descending ? desc : asc;
-          });
           this.setState(
-            { data: sortedData, sort },
+            { sort },
             () => { this.handlePageChange(1) }
           );
         }
 
         handlePageChange(currentPage) {
           this.setState({ currentPage })
         }
+        
+        // See 'sortable columns — controlled' example for more implementation details
```

Comment:
I would capitalize here to match the title of the example (and maybe direct to "above" since we can't add a link)